pydantic/_internal/_config.py

```diff
@@ -273,7 +273,7 @@ def push(self, config_wrapper: ConfigWrapper | ConfigDict | None):
     ser_json_inf_nan='null',
     validate_default=False,
     validate_return=False,
-    protected_namespaces=('model_',),
+    protected_namespaces=('model_validate', 'model_dump'),
```

Comment:
Do we also want to add `model_construct` here?

Reply:
Good question, cc @samuelcolvin @dmontagu

Reply:
Chatted with DM, I think no for now.

Reply:
Following up here - we decided we didn't need to add `model_construct` - that's not a namespace we anticipate expanding...